packages/web/docs/src/pages/docs/gateway/deployment/runtimes/bun.mdx

```diff
@@ -13,7 +13,16 @@ platform besides Node.js. [Bun](https://bun.sh) is a modern JavaScript runtime l
 and it supports Fetch API as a first class citizen. So the configuration is really simple like any
 other JS runtime with Hive Gateway;
 
-The following code is a simple example of how to use Hive Gateway with Bun.
+## Hive Gateway CLI
+
+You can follow the introduction page directly to [use Hive Gateway CLI](/docs/gateway#installation).
+
+Since Bun has the compatibility layer for Node.js, all Node specific features are available in Bun.
+
+## Hive Gateway Runtime (advanced-only)
```

Comment:
Why do we claim that the programmatic API is for advance usecases an only to be used if "you know what you're doing"?

Its completely ok and really not that advanced to use the prog. API.

Reply:
I copied what we have in Node.js page

Reply:
Removed (Advanced)